### DISCUSSION TITLE
Filament type

### DISCUSSION THREAD

**Discussion by DiscusZ** [Donkie/Spoolman#468] [Q&A] [answered]
Where doees Spoolan get the type of filament?   I use Prusa and  I get conflicts with PETG as in prusa it is set to PET, also  I have PLA+  but no option in prusa for that either.  I save y profiles as PLA+  PETG   PLA-CF or the like

Thanks

**Comment by Donkie** [comment, marked as answer]
Filaments are sourced from this folder: https://github.com/Donkie/SpoolmanDB/tree/main/filaments
